syntax = "proto3";

package example;

option cc_generic_services = true;

message ResultCode
{
  int32 code = 1;
  string msg = 2;
}

message LoginRequest
{
  string name = 1;
  string pwd = 2;
}

message LoginResponse
{
  ResultCode result = 1;
  bool success = 2;
}

service UserServiceRpc
{
  rpc Login(LoginRequest) returns(LoginResponse);
}